### ISSUE TITLE
Bootstrap 4 - css file

### ISSUE THREAD

**Issue by mmjaeger** [twbs/bootstrap#19416]
Hello
Had a look thru the css file and I was surprised to find a lot repetition of code:

`.col-xs-1 {
  position: relative;
  min-height: 1px;
  padding-right: 15px;
  padding-left: 15px;
  float: left;
  width: 8.333333%;
}
`

we have 12 columns and maybe three or for breakpoints - for each column size we are adding:
`position: relative;
  min-height: 1px;
  padding-right: 15px;
  padding-left: 15px;
  float: left;`

wouldn't it make more sense to maybe add an additional class like 'column' or 'columns' and set this only once like:
`.columns {
position: relative;
  min-height: 1px;
  padding-right: 15px;
  padding-left: 15px;
  float: left;
}`

then we would only need to add this per column size and breakpoint:
`.col-xs-1 {
  width: 8.333333%;
}
`

**Comment by cvrebert** [comment]
Approximate duplicate of #17228.
